x-pack/qa/sql/security/build.gradle

```diff
@@ -51,6 +54,9 @@ subprojects {
     setting 'xpack.license.self_generated.type', 'trial'
     // Setup roles used by tests
     extraConfigFile 'roles.yml', '../roles.yml'
+    // Change the log4j configuration for audit logging file, more specifically make it roll over every 2 days
+    // Some tests do read the audit log file and do not account for the daily (default) rollover
+    extraConfigFile 'log4j2.properties', '../log4j2.properties'
```

Comment:
Could we have a task to copy and modify the original from `distributions/src/config` ? 
You can reference the location as `"${project(':distributions').projectDir}/src/config/log4j2.properties`, have a copy task to copy this one file and then have a `doLast` to modify it as you would like.